### ISSUE TITLE
Saving graph layout for later use

### ISSUE THREAD

**Issue by sempervivo** [sameeraroshan/visjs#18]
Is there a possibility to save all the nodes' positions so that when the same user uses the app again, thew ill be loaded in the same positions he arranged them the last time ?

Thank you.

**Comment by sameeraroshan** [comment]
Yes. You can use get seed method to save the seed. Provide saved seed for the next diagram and you will get the same positions.  Please read the vus js documentation. 

Sent from my Samsung Galaxy smartphone.

-------- Original message --------

**Comment by sempervivo** [comment]
Sorry. I posted in the wrong forum. I was referring to your Vaadin visjs addon. I didn't see any options to save the seed of the network diagram class. :(

**Comment by sameeraroshan** [comment]
ill add that and release a new version.

**Comment by sempervivo** [comment]
Great. Thank you.

**Comment by wangzhenux** [comment]
Hi @sameeraroshan , it’s great to hear you’re adding this feature. Have you already added it? If so, is there any documentation? If not, do you have an expected timeframe?

Thank you!
